File: CS50/Problems/sentimental-mario-less/mario.py
from cs50 import get_int
# height
N = 0
while N < 1 or N > 8:
    N = get_int("Height: ")

# rows
for i in range(N):
    # columns
    for j in range(N, i + 1, -1):
        print(" ", end="")
    # bricks    
    for q in range(-1, i, 1):
        print("#", end="")
    print("")    

File: CS50/Problems/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get num between 1 and 8.
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    // rows
    for (int i = 0; i < n; i++)
    {
        // spaces
        for (int d = n; d > i + 1; d--)
        {
            printf(" ");
        }
        // bricks
        for (int r = -1; r < i; r++)
        {
            printf("#");
        }
        
        printf("\n");
    }
}